presto-main/src/test/java/com/facebook/presto/sql/planner/TestTypeValidator.java

```diff
@@ -129,8 +130,8 @@ public void testValidProject()
         Expression expression1 = new Cast(new SymbolReference(variableB.getName()), StandardTypes.BIGINT);
         Expression expression2 = new Cast(new SymbolReference(variableC.getName()), StandardTypes.BIGINT);
         Assignments assignments = Assignments.builder()
-                .put(newVariable(variableAllocator, expression1, BIGINT), castToRowExpression(expression1))
-                .put(newVariable(variableAllocator, expression2, BIGINT), castToRowExpression(expression2))
+                .put(newVariable(variableAllocator, expression1, BIGINT), translator.translate(expression1, TypeProvider.fromVariables(ImmutableList.of(variableB))))
```

Comment:
static

Reply:
I believe its easier to read without static. I easily know the type of return value.